lib/csv_report_generator.rb

```diff
@@ -2,7 +2,7 @@
 require 'redis-lock'
 
 class CsvReportGenerator
-  CSV_PATH = "#{Rails.root}/reports"
+  CSV_PATH = "/var/apps/publisher/reports"
```

Comment:
It would be better to use `#{ENV['GOVUK_APP_ROOT'] || Rails.root}` here.  That way it will work in a non-deployment environment.  It'll also be resilient to changes in the filesystem layout on the servers.